clipboard/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@capacitor/clipboard",
+  "version": "0.0.1",
+  "description": "The Clipboard API enables copy and pasting to/from the clipboard.",
+  "main": "dist/plugin.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "scripts": {
+    "lint": "npm run prettier -- --check && npm run swiftlint -- lint",
+    "fmt": "npm run prettier -- --write && npm run swiftlint -- autocorrect",
+    "prettier": "prettier \"**/*.{css,html,ts,js,java}\"",
+    "swiftlint": "node-swiftlint",
+    "build": "npm run clean && tsc && rollup -c rollup.config.js",
+    "clean": "rimraf ./dist",
+    "watch": "tsc --watch",
+    "prepublishOnly": "npm run build"
+  },
+  "author": "Ionic <hi@ionicframework.com>",
+  "license": "MIT",
+  "devDependencies": {
+    "@capacitor/android": "next",
+    "@capacitor/core": "next",
+    "@capacitor/ios": "next",
+    "@ionic/prettier-config": "^1.0.0",
+    "@ionic/swiftlint-config": "^1.0.0",
+    "@rollup/plugin-node-resolve": "^8.1.0",
+    "prettier": "^2.0.5",
+    "prettier-plugin-java": "^0.8.0",
+    "rimraf": "^3.0.0",
+    "rollup": "^2.21.0",
+    "swiftlint": "^1.0.1",
+    "typescript": "~3.9.7"
+  },
+  "peerDependencies": {
+    "@capacitor/core": "^2.4.0"
```

```suggestion
    "@capacitor/core": "^3.0.0-alpha.0"
```

we shouldn't use 2.x as peerDependency, should be the alpha or next (suggested alpha because the only merged plugin used that)